arelle/plugin/validate/ESEF/__init__.py

```diff
@@ -132,10 +132,12 @@ def modelDocumentPullLoader(
                         return None  # allow zipped test case to load normally
 
                 isZipFormat = modelXbrl.fileSource.isZip
-                hasZipFileExtension = modelXbrl.fileSource.type.lower() == ".zip"
+                hasZipFileExtension = modelXbrl.fileSource.type.lower() == ".zip" or (modelXbrl.fileSource.type == ".xbri" and  disclosureSystemYear >= 2024)
                 if disclosureSystemYear >= 2023 and not (isZipFormat and hasZipFileExtension):
-                    modelXbrl.error("ESEF.2.6.1.disallowedReportPackageFileExtension",
-                                    _("A report package MUST conform to the .ZIP File Format Specification and MUST have a .zip extension."),
+                    errorMessage = _("A report package MUST conform to the .ZIP File Format Specification and MUST have a .zip{} extension.")
+                    errorMessage = errorMessage.format(_(" or .xbri") if disclosureSystemYear >= 2024 else "")
```

Comment:
I'm not confident that sentence fragments like this can be translated correctly.

Reply:
Do you prefer to have 2 separate sentences?  I did that to avoid having two nearly equals messages to translate, I guessed that the sentence structure for an "or" shouldn't change too much between language, but I might be wrong.

Reply:
I have no preference.  I understood your intent, but my understanding from previous training is that it's best to avoid sentence fragments since no matter how "simple" they appear to be they usually cause problems for some language translation.  I suppose we can revisit if someone attempts that and reports problems.